src/performance/paint.ts

```diff
@@ -0,0 +1,14 @@
+import { Event } from "@clarity-types/data";
+import { PaintState } from "@clarity-types/performance";
+import encode from "./encode";
+
+export let state: PaintState = null;
```

Comment:
For each of these performance entry events could you add a link to the w3c spec you based it on? Fore xampel, this one is:
[https://www.w3.org/TR/paint-timing/](https://www.w3.org/TR/paint-timing/)

Reply:
Good point, adding.

---
In reply to: [347587865](https://github.com/microsoft/clarity-js/pull/160#discussion_r347587865) [](ancestors = 347587865)